Samples/UIWidgetsSamples_2019_4/Assets/Editor/EditorWindowSample/GalleryMain.cs

```diff
@@ -1,13 +1,29 @@
 using System.Collections.Generic;
 using UIWidgetsGallery.gallery;
 using Unity.UIWidgets.Editor;
+using Unity.UIWidgets.engine2;
 using Unity.UIWidgets.widgets;
 using UnityEditor;
+using UnityEngine;
 
 namespace UIWidgetsEditorWindowSample
 {
     public class EditorWindowGallery : UIWidgetsEditorPanel
     {
+        [MenuItem("UIWidgets/SetDebugLog")]
```

Comment:
put this menuitem inside the class D.

store the state inside D (e.g., D._enableDebug), instead of in UIWidgetsPanel.ShowDebugLog .

we need to tell the user the state too. try this; https://answers.unity.com/questions/775869/editor-how-to-add-checkmarks-to-menuitems.html